docs/reference/rest-api/index.asciidoc

```diff
@@ -57,9 +57,11 @@ not be included yet.
 * <<transform-apis,{transform-cap} APIs>>
 * <<usage-api,Usage API>>
 * <<watcher-api,Watcher APIs>>
+* <<rest-api-root,Root API>>
```

Comment:
This will need to be alphabetically reordered since renaming from Version API to Root API :)

Reply:
done, cheers